### DISCUSSION TITLE
Responsive design

### DISCUSSION THREAD

**Discussion by falbertoram** [reflex-dev/reflex#2852] [Q&A]
Is there any guide in relation to effective responsive techniques that any programmer should take into account?? I mean in the context of Reflex as I'm not sure about how Reflex handles responsive design.

**Comment by picklelo** [comment]
We have this section on our docs: https://reflex.dev/docs/styling/responsive/#responsive

Currently responsive props work with any CSS prop, but don't work with component-specific props. This is something we're working to add in the future.
